katalog/nginx-gke/README.md

```diff
@@ -24,7 +24,28 @@ Fury distribution NGINX GKE is deployed with following configuration:
 
 ## Deployment
 
-You can deploy NGINX GKE by running following command in the root of the project:
+Our Ingress module by default is deployed as a `DaemonSet`, meaning that it will try to deploy 1 ingress-controller pod on every worker node.
```

Comment:
Mixed feelings on putting this duplicated information on gke, ovh, and so on, we should link the main module README as additional architectural informations

Reply:
I'm not a fan either, but we have most of the readme duplicated anyway